src/collectors/diskusage/diskusage.py

```diff
@@ -63,7 +63,8 @@ def get_default_config(self):
                          '|sd[a-z]+[0-9]*$' +
                          '|x?vd[a-z]+[0-9]*$' +
                          '|disk[0-9]+$' +
-                         '|dm\-[0-9]+$'),
+                         '|dm\-[0-9]+$' +
+                         '|nvme[0-9]+[a-z]+[0-9].*$'),
```

Comment:
I don't believe so. Here's the naming convention of the devices: nvme0n1, nvme0n1p1, nvme0n1p2